docs/UserGuide.md

```diff
@@ -2,23 +2,33 @@
 layout: page
 title: User Guide
 ---
-
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
-
-* Table of Contents
-{:toc}
+## ConcertAddressBook
+ConcertAddressBook is a **desktop app** for **Concert Organisers** to manage your Concert contacts, optimized for use via a Command Line Interface (CLI) with also a Graphical User Interface (GUI) in place to assist with information display. Manage your _contacts_ and _concerts_ in the book, finding contact information faster than your regular GUI contact management applications.
+
+1. [Quick start](#Quick-Start)
+2. [Features](#features)
+   1. [Adding a Person](#adding-a-person-addp)
+   2. [Listing all persons](#listing-all-persons--list)
+   3. [Editing a person](#editing-a-person--edit)
+   4. [Finding a person](#finding-a-person-findp)
+   5. [Deleting person](#deleting-a-person--delete)
+   6. [Clearing All Entries](#clearing-all-entries--clear)
+   7. [Exiting programme](#exiting-the-program--exit)
+3. [FAQ](#faq)
+4. [Known Issues](#known-issues)
+5. [Command Summary](#command-summary)
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
 1. Ensure you have Java `17` or above installed in your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `.jar` file from [here](nil). (COMING SOON)
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your ConcertAddressBook.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
+1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar tp.jar` command to run the application.<br>
```

Comment:
go to build.gradle to change the name from addressbook to concertphonebook `line 69` then test with `./gradlew clean shadowjar` and check inside `build/libs` for the jar file should be changed to the new name